openshift-java/Jenkinsfile

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env groovy
+
+node('rhel7'){
+	stage('Checkout repo') {
+		deleteDir()
+		git url: 'https://github.com/redhat-developer/vscode-openshift-extension-pack/'
+        dir: 'openshift-java'
+	}
+
+	stage('Install requirements') {
+		def nodeHome = tool 'nodejs-8.11.1'
+		env.PATH="${env.PATH}:${nodeHome}/bin"
+		sh "npm install -g typescript vsce"
+	}
+
+	stage('Build') {
+		sh "npm install"
+		sh "npm run build"
+	}
+
+	withEnv(['JUNIT_REPORT_PATH=report.xml']) {
+        stage('Test') {
+    		wrap([$class: 'Xvnc']) {
+    			sh "npm test --silent"
+    			junit 'report.xml'
+    		}
+        }
+	}
+
+	stage('Package') {
+		try {
+			def packageJson = readJSON file: 'package.json'
+			sh "vsce package -o adapters-${packageJson.version}-${env.BUILD_NUMBER}.vsix"
```

Comment:
The vsix file name match extension pack name.

Reply:
Fixed this.